test/it/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.UnitSpec
+import config.AppConfig
+import org.scalatest.GivenWhenThen
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.test.Injecting
+import testdata.TestData
+
+class CustomsDataStoreConnectorSpec extends UnitSpec with GivenWhenThen with GuiceOneAppPerSuite with Injecting {
+
+  implicit val appConfig: AppConfig = inject[AppConfig]
+
+  "CustomsDataStoreConnector.url" should {
```

Comment:
I can see there are 2 suites testing this connector. I'd suggest merging them into one. In my opinion integration suite is more appropriate in this case.
You could use `WireMock` for this to test if connector returns the response correctly. In the same time, the url used by the connector can be asserted.